types/jsreport-core/tslint.json

```diff
@@ -1 +1,9 @@
-{ "extends": "dtslint/dt.json" }
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"no-const-enum": false,
+		"no-declare-current-package": false,
+		"no-mergeable-namespace": false,
+		"strict-export-declare-modifiers": false
```

Comment:
Please enable these rules again.

Reply:
I do want disable rule `no-const-enum` because in this module, actually there is no enum exported, but at the same time, I want use enum because after tsc compiled, it would changed into string value in js.  I don't want use string value as the type because I want extend them in other model. just like the interface `I` in the example below.
module a
```ts
declare namespace A {
  const enum Type{
    a = 'typeA';
  }
}
declare function A(type: A.Type): void;
declare module 'a' {
  export = A;
}
```

module b
```ts
declare namespace A {
  const enum Type{
    b = 'typeB';
  }
}
declare namespace B {
}
declare function B(): void;
declare module 'b' {
  export = B;
}
```